src/Theme/ThemeColorExtractor.php

```diff
@@ -0,0 +1,85 @@
+<?php
+
+namespace Phiki\Theme;
+
+class ThemeColorExtractor
```

Comment:
Is there any way we can put this behaviour on the `ParsedTheme` class and use the same `TokenColor` object that we've already got elsewhere for consistency?